frontend/src/Pages/MeetTheTeamPage/MeetTheTeamPage.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Col from 'react-bootstrap/Col';
+import Nav from 'react-bootstrap/Nav';
+import Row from 'react-bootstrap/Row';
+import Tab from 'react-bootstrap/Tab';
+import Button from 'react-bootstrap/Button';
+
+const MeetTheTeamPage = () => {
+    const teamMembers = [
+      {
+        name: 'Maryellen Marino',
+        role: 'President',
+        avatar: '', 
```

Comment:
Sending you both everyone's avatar now. I'll also send over Jr Directors shortly and after that we can also add Project Leads and potentially Organizers.